### ISSUE TITLE
tsh login behavior with Kubernetes

### ISSUE THREAD

**Issue by kontsevoy** [gravitational/teleport#2545]
## Problem

Teleport users are uhappy with `tsh` modifying their `kubectl` and are looking for an obvious way to turn this off.

## Proposals

* If K8s integration is turned off on the server, `tsh` should never touch `kubeconfig`.
* If K8s integration is turned on, the default behaviour is to update `kubeconfig`.
* Add a way to turn kubeconfig integration OFF via a tsh profile file.
* Re-assign this to Ev to update documentation

## Profile file

Currently `tsh` creates a profile (config) file for every proxy server it connects to, i.e. 

```bash
$ tsh login --proxy=example.com
```

... will create `~/.tsh/example.com.yaml` file which looks like this:

```yaml
web_proxy_addr: example.com:3080
ssh_proxy_addr: example.com:3023
user: kontsevoy
```

Let's add another value to this file, which users should be able to edit:

```yaml
# by default (when missing) both values are assumed
certificates: ['ssh', 'k8s']
```

...so, to disable kubernetes, a user would set:

```yaml
certificates: ['ssh']
# or...
certificates: ssh
```

**Comment by kontsevoy** [comment]
@klizhentas TBD.

**Comment by klizhentas** [comment]
Lets move this one out of the milestone,as we talked about implementing this as a user profile. This is partially works, when K8s integration is off, user kubeconfig will not be modified.

**Comment by russjones** [comment]
Removing from 3.1.6 milestone as per discussion.

**Comment by awly** [comment]
Closing in favor of https://github.com/gravitational/teleport/issues/6045
Will be fixed in 6.2
